hack/README.md

```diff
@@ -53,14 +53,18 @@ The standard workflow is performed inside a helper container to normalize the bu
 - `cluster-up`: start a default Kubernetes or Open Shift cluster. set KUBEVIRT_PROVIDER environment variable to either 'k8s-1.18' or 'os-3.11.0-crio' to select the type of cluster. set KUBEVIRT_NUM_NODES to something higher than 1 to have more than one node.
 - `cluster-down`: stop the cluster, doing a make cluster-down && make cluster-up will basically restart the cluster into an empty fresh state.
 - `cluster-down-purge`: cluster-down and cleanup all cached images from docker registry. Accepts [make variables](#make-variables) DOCKER_PREFIX. Removes all images of the specified repository. If not specified removes localhost repository of current cluster instance.
-- `cluster-sync`: builds the controller/importer/cloner, and pushes it into a running cluster. The cluster must be up before running a cluster sync. Also generates a manifest and applies it to the running cluster after pushing the images to it.
+- `cluster-sync`: 'make cluster-sync-cdi' followed by 'make cluster-sync-test-infra'
+- `cluster-sync-cdi`: builds the controller/importer/cloner, and pushes it into a running cluster. The cluster must be up before running a cluster sync. Also generates a manifest and applies it to the running cluster after pushing the images to it.
     - `cluster-sync-controller`: builds the controller and pushes it into a running cluster. 
```

Comment:
these aren't targets, right? so we can just remove these lines.
Happy to just do this in a separate PR though.

Reply:
which ones are not targets?

Reply:
cluster-sync-importer, cluster-sync-cloner, etc.

Reply:
please do it in separate PR, as I see there are many other dead targets as well.

Reply:
yes, many more :), and true, it can be done in other PR